File: Selenium/Activities/Activity1.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[]args){
        // Set up Firefox driver
        WebDriverManager.firefoxdriver().setup();
        // create a new instance of firefox driver
        WebDriver driver = new FirefoxDriver();

        // open the page
        driver.get("https://v1.training-support.net/");
        //print the title page
        System.out.println("Page title is " + driver.getTitle());
        //FInd the button element
        WebElement element = driver.findElement(By.id("about-link"));
        element.click();
        //navigate to the new page
        System.out.println("About Page title is " + driver.getTitle());


        driver.quit();
    }
}

File: Selenium/Activities/Activity1.py
# Import selenium from webdriver
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up firefox driver with Webdriver manager
service = FirefoxService(GeckoDriverManager().install())
# Start the driver
with webdriver.Firefox(service=service) as driver:
    driver.get("https://v1.training-support.net/")
    # Page title
    print("Page title is: " + driver.title)
    # Find about us button and click
    driver.find_element(By.ID, "about-link").click()
    # New page title
    print("New page title is: " + driver.title)
